internal/build/templates/ensure-sysext.service

```diff
@@ -9,12 +9,16 @@ ConditionDirectoryNotEmpty=|/run/extensions
 ConditionDirectoryNotEmpty=|/var/lib/extensions
 ConditionDirectoryNotEmpty=|/usr/local/lib/extensions
 ConditionDirectoryNotEmpty=|/usr/lib/extensions
+ConditionPathExists=/usr/bin/elemental3ctl
 
 [Service]
 Type=oneshot
 RemainAfterExit=yes
 ExecStart=/usr/bin/systemctl daemon-reload
+# TODO (atanasdinov): Investigate a SELinux limitation where elemental3ctl cannot be called directly here
+ExecStart=/bin/sh -c "/usr/bin/elemental3ctl kmod --reload"
```

Comment:
can we move this to its own systemd service rather than overloading `ensure-sysext` with it? as I fixed the reload manager handling (with vojtechs help), this entire service should actually be removeable..

Reply:
We can do the cleanup in a follow-up, though!